db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.3.0_upgrade.sql

```diff
@@ -1,21 +1,25 @@
 -- Upgrade script for master.app_detail
+DELETE FROM master.app_detail WHERE lang_code !='eng';
```

Comment:
We are deleting data in upgrade sql, but on failure how are we planning to restore without backup? @dhanendra06

Reply:
created a backup table @ase-101